docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,146 @@
+| Nowra ADC    | | Nowra Tower  | 118.850         | NW_TWR        |
+| Nowra SMC   ||  Nowra Ground  | 135.850         | NW_GND        |
+| Nowra ACD    | | Nowra Delivery  | 128.350         | NW_DEL       |
+| Nowra ATIS    |   | | 125.65         | YSNW_ATIS       |
+
+## Airspace
+
+### TMA Ownership
+
+The Terminal Control Area (TMA) owns the airspace within R421 up to FL125; optionally R420F extends from FL125 to FL300.
+This airspace is for facilitating military aircraft and low-level helicopter operations.
+
+### Activation of Restricted Areas
+
+When a controller logs in as Nowra ADC (NW ADC) or TMA:
+
+1. **Announce Activation**: Announce that R421 and other applicable PRDs are active.
+2. **Activate Surrounding Areas**: Notify surround controllers to activate the restricted areas to get the outlines visible.
+3. **Issue Clearances**: Treat restricted areas like R421 as Class C/D airspace and issue clearances accordingly.
+4. **Service Levels**:
+    - Official aircraft: Class D service. (eg Military)
+    - Unofficial aircraft: Class C service. (eg Non Military)
+
+### Airspace Details
+
+<figure markdown>
+![NW ADC Airspace](img/nwadc.png){ width="700" }
+<figcaption>NW ADC Airspace</figcaption>
+</figure>
+
+
+- **R421**: Extends up to FL125 and is used for military and low-level helicopter operations.
+- **R420F**: Extends from FL125 to FL300.
+
+## VFR corridor (DAY ONLY)
+<figure markdown>
+![NW VFR CRD](img/nwvfc.png){ width="700" }
+<figcaption>NW VFR CRD</figcaption>
+</figure>
+- All VFR aircraft passing through R421 must request clearance and be assigned a squawk code.
+- If clearance is not immediately available, instruct the aircraft to remain in Class G airspace.
+- VFR traffic must not pass through the corridor above 2000FT AMSL.
+- Separation must be maintained between IFR and VFR traffic including providing necessary traffic advisories.
+
+## Aerodrome
+
+### General Guidelines
+
+1. **Ground Taxiing for Wheeled Helicopters**
+    - Where practicable, wheeled helicopters are encouraged to “ground taxi” on prepared surfaces to minimize rotor wash and its effects.
+
+### The Runway Mode options available are:
+
+a) Runway 21 and Runway 08;  
+b) Runway 21 and Runway 26;  
+c) Runway 21 only;  
+d) Runway 03 and Runway 08;  
+e) Runway 03 and Runway 26;   
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Miscellaneous
+### Circuit Procedures
+Circuit operations at Nowra occur within the CIRA, a `5NM` radius of Nowra, from the surface to `2000ft`.
+
+- Fixed wing: `2000ft`
+- Rotary wing: `1200ft`
+- Flight Deck Procedural Trainer: not above `700ft`
```

Comment:
Change `2000ft` to `A020`, etc (for standardisation with other SOP pages)

Reply:
Darwin etc is in this format. Direct copy paste so probably worth revisiting for those pages.